includes/Embed_Block.php

```diff
@@ -132,85 +135,32 @@ public function render_block( array $attributes, $content ) { // phpcs:ignore Va
 			$attributes['title'] = __( 'Web Story', 'web-stories' );
 		}
 
-		if ( is_feed() ) {
-			return $this->render_block_feed( $attributes );
-		}
+		$defaults = [
+			'align'  => 'none',
+			'height' => 0,
+			'poster' => '',
+			'width'  => 0,
+		];
 
-		return $this->render_block_html( $attributes );
-	}
+		$attributes = wp_parse_args( $attributes, $defaults );
 
-	/**
-	 * Renders the block type output in default context.
-	 *
-	 * @param array $attributes Block attributes.
-	 *
-	 * @return string Rendered block type output.
-	 */
-	protected function render_block_html( array $attributes ) {
-		$url          = (string) $attributes['url'];
-		$title        = (string) $attributes['title'];
-		$poster       = ! empty( $attributes['poster'] ) ? esc_url( $attributes['poster'] ) : '';
-		$align        = sprintf( 'align%s', $attributes['align'] );
-		$margin       = ( 'center' === $attributes['align'] ) ? 'auto' : '0';
-		$player_style = sprintf( 'width: %dpx; height: %dpx; margin: %s', absint( $attributes['width'] ), absint( $attributes['height'] ), esc_attr( $margin ) );
-		$poster_style = ! empty( $poster ) ? sprintf( '--story-player-poster: url(%s)', $poster ) : '';
-
-		if ( ! function_exists( 'is_amp_endpoint' ) || ! is_amp_endpoint() ) {
-			wp_enqueue_style( 'standalone-amp-story-player' );
-			wp_enqueue_script( 'standalone-amp-story-player' );
+		$data = [
+			'title'            => $attributes['title'],
+			'url'              => $attributes['url'],
+			'poster_portrait'  => $attributes['poster'],
+			'poster_square'    => $attributes['poster'],
+			'poster_landscape' => $attributes['poster'],
```

Comment:
That doesn't make sense.

`$attributes['poster']` is always the portrait poster, so only the portrait poster should be set.

`poster_landscape` and `poster_square` should not be set to a portrait poster.

```suggestion
			'title'           => $attributes['title'],
			'url'             => $attributes['url'],
			'poster_portrait' => $attributes['poster'],

```